.github/workflows/ci.yml

```diff
@@ -82,6 +82,12 @@ jobs:
 
   tag-semver:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
Wouldn't using this matrix defeat the purpose of disabling latest since it would just do a run where it was true?

if you agree, I can submit a PR that instead of using a matrix, it uses a conditional expression for `tag-latest` instead of this matrix example. 

Here's a in-production example of how I'm using it: https://github.com/OctoPrint/octoprint-docker/blob/master/.github/workflows/octoprint-release.yml#L75-L76

Or I could port the example from the docs of HackerHappyHour/tagging-strategy ([Conditionally produce 'latest'](https://github.com/HackerHappyHour/tagging-strategy#conditionally-produce-latest))

Reply:
@LongLiveCHIEF The ci workflow in this repo is just for testing purpose. Some sort of integration test if you want.